### DISCUSSION TITLE
Support for booklets

### DISCUSSION THREAD

**Discussion by martensson** [navidrome/navidrome#2551] [Ideas]
Hi! 

Starting to use Navidrome more and more and really like the project so far. I had a feature request that would be great to have, and that is to include a button to download/view booklets in the album view. A lot of people scan the CD album booklet, and online services like Qobuz, Deezer, etc., provide a digital pdf booklet with the album. 

The implementation could be as simple as adding a booklet button in case there is a "booklet.pdf" inside the album dir. It would also be great to include it when downloading the entire album (also the cover.jpg that is not included right now).

Thanks in advance!

**Comment by arg274** [comment]
Second this. I think the existing lightbox being used for showing the album cover might be a better target to show all the scans/artwork. (Roon is the only player I can think of that does the same)

**Comment by deluan** [comment]
Will keep it in mind for the new UI.

c/c @caiocotts

**Comment by wie-was** [comment]
I just want to second this. Navidrome has already become the central hub for my digital music collection, and one thing that's missing when you compare it to the delight of picking a CD or Vinyl from the shelf is the missing booklet and back/internal side of the artwork.

I noticed that on musicbrainz you often find such information, eg. https://musicbrainz.org/release/bebaff77-6f7d-42cf-b3d2-6cfaa4b94f47/cover-art (just a kinda unrelated observation).

**Comment by mist2t** [comment]
I'm also a supporter for this feature. 

It will be super cool to view those scans (cd booklets). 

Even if i have to properly rename / organize all of them to a naming convention, i would do it 🙂👍

**Comment by nozonyan** [comment]
Are there any updated plans for this? As an early music enjoyer it is often imperative to follow along with booklet (even in case of archaic versions of one 's native language, eg middle ages german/french, let alone when dealing with polyphonic bi and trilingual pieces) to make sense of the music altogether. Having even a download booklet button would be superhelpful.

**Comment by lukebemish** [comment]
This would be an awesome thing to have! Even if apps consuming the API don't support it immediately just having the info available through the API would be super useful so it could even be implemented on that end.

**Comment by ImAsra** [comment]
this would also be cool for game soundtracks adding the artwork booklet to the albums so i +1 this too
